deploy/cdk/queue_stack.py

```diff
@@ -56,6 +56,24 @@ def __init__(
             )
         )
 
+        self.vector_queue = self._queue(
+            f"{construct_id}-vector-queue",
+            visibility_timeout=900,
+            dead_letter_queue=sqs.DeadLetterQueue(
+                max_receive_count=5,
+                queue=self._queue(f"{construct_id}-vector-dlq"),
+            ),
+        )
+
+        lambda_stack.trigger_vector_lambda.add_event_source(
+            lambda_event_sources.SqsEventSource(
+                self.vector_queue,
+                batch_size=1,
```

Comment:
might want to increase the `batch_size`?

Reply:
Good catch. Will increase to match the other queues.